classes/04class/exercises/c04-iac01/carloshz4/_outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "ip-instance" {
+  value = "${aws_instance.da_academy.public_ip}"
```

Comment:
> Fantastic job @carloshz4
> 
> You are doing well with the exercises. Kudos for that ✨✨
> 
> Just left a few notes for you in this one. Let me know what do think 🤗

Thanks mate for your words of encouragement and suggestions.  I'll have a look.

Reply:
> This line will generate the warning: `Warning: Interpolation-only expressions are deprecated`
> 
> Use interpolation when you need to concat the variables like: `"my-var-${var.my_random_number}"`. Otherwise, just use the variable as it is (without interpolation).
> 
> Tip: you can run the following command to upgrade your code `terraform 0.12upgrade`. Have a look at the changes that it will do. 😉

I may have overlooked that warning, not sure.  I have version v0.12.12. Thanks for the suggestion, I'll look into it.

Reply:
Would you like me to resubmit the PR with your suggestions?

Reply:
I reckon that it will be best to avoid future students to get confuse with that.
Remember that we are doing the exercises for ourselves and to help anyone which will like to learn more about.

Reply:
> I reckon that it will be best to avoid future students to get confuse with that.
> Remember that we are doing the exercises for ourselves and to help anyone which will like to learn more about.

Good point, better to do it properly 👍